en/PlanView/pattern_fixed_wing_landing.md

```diff
@@ -0,0 +1,60 @@
+# Fixed Wing Landing Pattern (Plan Pattern)
```

Comment:
1. Does this work for both ArduPilot and PX4 flight stacks (ie is it independent of their landing mechanisms)?
1. Does this obey any of the flight stack behaviour for flare etc? I.e. how would you describe the pattern?

Reply:
Works on all stacks. Has no influence on flare which is controlled by firmware. Firmware should complain in upload if mission violates flare constraints.

Reply:
@DonLakeFlyer I've merged this, but want to understand this a bit better to see if it needs further clarification. 

Specifically, say on PX4 you have a Land "mode" and some [land behaviour as defined by parameters](https://docs.px4.io/en/flying/fixed_wing_landing.html). Because you say this works on all autopilots and seems to differ from the above pattern, I'm guessing that you're not just putting into Land mode?

So how does it work? More specifically, I'm wondering how the firmware knows it is landing so it can complain about flare constraints being broken.

Reply:
It creates three mission items:
* DO_LAND_START - If you abort a landing it sends DO_GO_AROUND to the vehicle which then causes the mission to return to this point and try to land again.
* NAV_LOITER_TO_ALT - Start point for landing
* NAV_LAND - End point for panding

These mission commands are supported by both firmwares. The vehicle then flares to landing using a flight path generated by the firmware between the NAV_LOITER_TO_ALT point and the NAV_LAND point. 

When you upload a mission to the vehicle if those two locations violate the flare contrainsts you will get an error (like say descent angle is too steep). On PX4 it sends an error message to the ground station at upload time. And it will refuse to start the mission since it fails integrity checks. Not sure how ArduPilot handle that error case. There are multiple ways to do it.